ollama/deployment/src/main/java/io/quarkiverse/langchain4j/ollama/deployment/OllamaProcessor.java

```diff
@@ -47,58 +53,98 @@ public void providerCandidates(BuildProducer<ChatModelProviderCandidateBuildItem
         }
     }
 
-    @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+    @BuildStep(onlyIfNot = IsNormal.class, onlyIf = Langchain4jDevServicesEnabled.class)
+    public void devServicesSupport(List<SelectedChatModelProviderBuildItem> selectedChats,
+            List<SelectedEmbeddingModelCandidateBuildItem> selectedEmbeddings,
+            LangChain4jOllamaFixedRuntimeConfig fixedRuntimeConfig,
+            BuildProducer<DevServicesChatModelRequiredBuildItem> chatProducer,
+            BuildProducer<DevServicesEmbeddingModelRequiredBuildItem> embeddingProducer) {
+        for (var selected : selectedChats) {
+            if (PROVIDER.equals(selected.getProvider())) {
+                String configName = selected.getConfigName();
+                String baseUrlProperty = String.format("quarkus.langchain4j.ollama%s%s",
+                        NamedConfigUtil.isDefault(configName) ? "." : ("." + configName + "."), "base-url");
+
+                // same as with other dev services, we only need it if the base URL has not been set by the user
+                if (!ConfigUtils.isPropertyPresent(baseUrlProperty)) {
+                    String modelId = NamedConfigUtil.isDefault(configName)
+                            ? fixedRuntimeConfig.defaultConfig().chatModel().modelId()
+                            : fixedRuntimeConfig.namedConfig().get(configName).chatModel().modelId();
+                    chatProducer.produce(new DevServicesChatModelRequiredBuildItem(PROVIDER, modelId, baseUrlProperty));
+                }
+            }
+        }
+
+        for (var selected : selectedEmbeddings) {
```

Comment:
Could we stick with the full term "embedding model" please? "Embeddings" is ambiguous (it can mean an embedding as in text+vector, or also embedding store). 
I know this happens in more places in the code...

Reply:
Sure, I'll fix it later

Reply:
Done